pkg/orchestration/wiring/wiringutil/knownshapes/scaling.go

```diff
@@ -0,0 +1,61 @@
+package knownshapes
+
+import (
+	smith_v1 "github.com/atlassian/smith/pkg/apis/smith/v1"
+	"github.com/atlassian/voyager/pkg/orchestration/wiring/wiringplugin"
+	"github.com/atlassian/voyager/pkg/orchestration/wiring/wiringutil/libshapes"
+)
+
+const (
+	// allows to select set of Kubernetes objects by list of scaling
+	SetOfScalingShape wiringplugin.ShapeName = "voyager.atl-paas.net/SetOfScalingShape"
```

Comment:
I believe we have a constant for prefix `voyager.atl-paas.net` and use it in many places already